src/app/files/basicReadOnlyExplorer.js

```diff
@@ -81,7 +81,18 @@ class BasicReadOnlyExplorer {
   // }
   //
   resolveDirectory (path, callback /* (error, filesList) => { } */) {
-    // path = '' + (path || '')
+    var self = this
+    if (path[0] === '/') path = path.substring(1)
+    var tree = {}
+    // This does not include '.remix.config', because it is filtered
+    // inside list().
+    Object.keys(this.list()).forEach(function (path) {
+      hashmapize(tree, path, {
+        '/readonly': self.isReadOnly(path),
+        '/content': self.get(path)
+      })
+    })
+    return callback(null, tree[path] || {})
```

Comment:
return `tree[path]` always undefined

Reply:
it's not always undefined.
sometimes it return `{ localhost: { foo: {}, bar: {}, baz: {} }`

...or ...hmm, maybe not? I think without it, the names like `gist`, `github`, etc... showed up in the file explorer

Reply:
well when I tested it with remixd it would always return `undefined`.
use publish to swarm to reproduce

Reply:
```I think without it, the names like gist, github, etc... showed up in the file explorer```
that's ok